packages/astro/templates/content/types.d.ts

```diff
@@ -109,6 +118,10 @@ declare module 'astro:content' {
 			? Promise<DataEntryMap[C][E]> | undefined
 			: Promise<DataEntryMap[C][E]>
 		: Promise<CollectionEntry<C> | undefined>;
+	export function getEntry<C extends keyof LiveContentConfig['collections']>(
+		collection: C,
+		filter: string | LiveLoaderEntryFilterType<C>,
```

Comment:
Why isn't `filter` optional here?

Reply:
This is getEntry, so you need to say which entry to get. It's either an id, or a query object to get by another field if the loader supports it